tests/cucumber/steps/index.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable no-console,global-require,no-loop-func,func-names */
 const assert = require('assert');
-const { Buffer } = require('buffer');
```

Comment:
This file only ever gets executed in node to set up the cucumber tests, so it would be perfectly acceptable to still use Buffer here

Reply:
Gotcha, reverted this back in [50e03cf](https://github.com/algorand/js-algorand-sdk/pull/801/commits/50e03cf358c119529a9bb4019962d9cabb4fe723)

Reply:
Also good point on removing the dependency -- I removed the browser buffer package and removed the lint rule around `Buffer` in [e7aa3d3](https://github.com/algorand/js-algorand-sdk/pull/801/commits/e7aa3d3f3694a82a67f69c837777198777ca70ae)